packages/runtime/container-runtime/src/summary/summaryFormat.ts

```diff
@@ -91,6 +91,8 @@ export interface IContainerRuntimeMetadata extends ICreateContainerMetadata, IGC
 	readonly summaryNumber?: number;
 	/** GUID to identify a document in telemetry */
 	readonly telemetryDocumentId?: string;
+	/** True if the runtime IdCompressor is enabled */
+	readonly idCompressorEnabled?: boolean;
```

Comment:
```suggestion
	readonly idCompressionEnabled?: boolean;
```

Reply:
I think `idCompressorEnabled` is more "correct" than `idCompressionEnabled`. I feel like `idCompressionEnabled` makes it sound like we're going to do something "automatically" for you whereas `idCompressorEnabled` makes it sound like we're giving a tool (the IdCompressor) that can be used.